timemachine/ff/__init__.py

```diff
@@ -1,11 +1,44 @@
+from pathlib import Path
+from typing import Union
+
+from timemachine import __file__ as tm_path
 from timemachine.ff.handlers import bonded, nonbonded
+from timemachine.ff.handlers.deserialize import deserialize_handlers
+
+PARAMS_DIR = Path(tm_path).parent / "ff" / "params"
 
 
 class Forcefield:
     """
     Utility class for wrapping around a list of ff_handlers
     """
 
+    @classmethod
+    def load_from_file(cls, path: Union[str, Path]) -> "Forcefield":
+        """Load a forcefield from a path
+
+        Parameters
+        ----------
+
+        path: string or pathlib.Path
+            Either the filename of a built in ff (smirnoff_1_1_0_sc.py) or a path to a new forcefield file
+
+        Returns
+        -------
+        Forcefield
+            Return a ForceField object constructed from parameters file
+        """
+        original_path = str(path)
+        path = Path(path)  # Safe to construct a Path object from another Path object
+        if not path.is_file():
+            # Search built in params for the forcefields
+            path = PARAMS_DIR / original_path
+        if not path.is_file():
+            raise ValueError(f"Unable to find {original_path} in file system or built in forcefields")
```

Comment:
nit: from an outside perspective this behavior (trying the file system first, then falling back to the built-in path) seems potentially a little surprising. E.g. is it possible to have a local file shadow a built-in, so it becomes tricky to figure out which version is being used?

Not a blocker, but wondering if this could be a little more explicit (e.g. having both `from_file` and `from_builtin` methods that share most of their implementation)

Reply:
Modified it in [edb97f9](https://github.com/proteneer/timemachine/pull/638/commits/edb97f939a10688fb3c4e2e2dc45fa2512ff200a) to print a warning and to fall back to the built in, as that is more likely to be trustworthy. Could also raise an exception, but in the moment I felt like just having a warning